ext/reflection/php_reflection.stub.php

```diff
@@ -710,8 +714,11 @@ public function __construct(object|string $class, string $constant) {}
 
     public function getEnum(): ReflectionEnum {}
 
-    /** @implementation-alias ReflectionClassConstant::getValue */
-    public function getValue(): UnitEnum {}
+    /**
+     * @return UnitEnum
+     * @implementation-alias ReflectionClassConstant::getValue
+     */
+    public function getValue(): mixed {} // TODO Fix return type
```

Comment:
What's happening here?

Reply:
I get a mysterious segfault if this tentative return type is added, so I excluded it for now. I'll debug this if the RFC is accepted.